test/e2e/main/main_test.go

```diff
@@ -1450,3 +1450,61 @@ func createCacheWithCR(cacheName string, nameSpace string, clusterName string) *
+	hostAddr, client := tutils.HTTPClientAndHost(ispn, testKube)
+	createCacheWithXMLTemplate(cacheName, hostAddr, template, client)
+
+	//Generate tons of random entries
+	for key := 1; key < 60000; key++ {
+		keyURL := fmt.Sprintf("%v/%v", cacheURL(cacheName, hostAddr), key)
+
+		headers := map[string]string{
+			"Content-Type": "text/plain",
+		}
+		resp, error := client.Post(keyURL, veryLongValue, headers)
+		defer tutils.CloseHttpResponse(resp)
+
+		if error != nil {
+			fmt.Printf("ERROR for key=%d, Description=%s", key, error)
+		}
+	}
+
+	//Check if all pods are running and they are not degraded
+	podList := &corev1.PodList{}
+	tutils.ExpectNoError(wait.Poll(tutils.DefaultPollPeriod, tutils.SinglePodTimeout, func() (done bool, err error) {
```

Comment:
You should also check restartCount==1 and pod is ready for pod that was hit by OOM. Other thing that would be nice to verify after this is condition is met is that no cache is in degraded state

Reply:
I have changed this part. Now it polls immediately after the OOM; whether the cluster is able to bootstrap again the test ends successfully. In every other case it fails